frontend/src/scenes/onboarding/personalizationLogic.ts

```diff
@@ -38,8 +39,25 @@ export const personalizationLogic = kea<personalizationLogicType<Personalization
                 number_of_answers: Object.keys(payload).length,
             })
             organizationLogic.actions.updateOrganization({ personalization: payload })
-
+        },
+        [organizationLogic.actionTypes.updateOrganizationSuccess]: async () => {
             window.location.href = '/'
         },
+        [organizationLogic.actionTypes.loadCurrentOrganizationSuccess]: async () => {
+            // Edge case in case this logic loaded before the api/organization request is completed
+            const personalization = organizationLogic.values.currentOrganization?.personalization
+            if (personalization && Object.keys(personalization).length) {
+                router.actions.push('/setup')
+            }
+        },
+    },
+    events: {
+        afterMount: () => {
```

Comment:
Well it doesn't make sense for a user to change these values. I'm particularly concerned that when we use these values to create the relevant demo data, changing them won't update the demo data and it will just lead to confusion.

Reply:
Right now demo data is created right after signup, before this gets submit. Of course that might change, but I think avoiding double-creating demo data should be handled in the backend rather than here.

Reply:
Well we'll definitely have to enable creating demo data after personalization as the result from that will be a key input for what demo data to generate. 

Definitely not suggesting to prevent double demo data creation just on the front-end, but I think it makes sense to disable this on the FE too. Maybe instead of a redirect we just do a message saying it's been completed like we do with the setup? That way the logic would end up being simpler.